src/core/config.py

```diff
@@ -45,47 +45,48 @@ def get_bool(setting: str, default: str = "False") -> bool:
 
 # Параметры логгера
 LOG_NAME = get_string("LOG_NAME")
+LOG_NAME = LOG_NAME if LOG_NAME is not None else "bot.log"
+
 LOG_PATH = BASE_DIR / "../logs" / LOG_NAME
 
 # Параметры локального сервера принимающего обновления от телеграм
 HOST = get_string("HOST", "0.0.0.0")
 WEBHOOK_URL = get_string("WEBHOOK_URL")
-PORT = get_int("BOT_PORT")
+
+PORT = 8000
+
 TOKEN = get_string("TELEGRAM_TOKEN")
 
 # Параметры для аутентификации телеграма
 SECRET_TELEGRAM_TOKEN = get_string("SECRET_TELEGRAM_TOKEN")
 
 # Параметры рассылки статистики
-WEEKLY_STAT_TIME = get_time("WEEKLY_STAT_TIME")
-WEEKLY_STAT_WEEK_DAYS = get_datetime_tuple("WEEKLY_STAT_WEEK_DAYS")
-MONTHLY_STAT_TIME = get_time("MONTHLY_STAT_TIME")
-MONTHLY_STAT_DAY = get_int("MONTHLY_STAT_DAY")
+WEEKLY_STAT_TIME = get_time("WEEKLY_STAT_TIME", "10:00")
+WEEKLY_STAT_WEEK_DAYS = get_datetime_tuple("WEEKLY_STAT_WEEK_DAYS", "0")
+MONTHLY_STAT_TIME = get_time("MONTHLY_STAT_TIME", "11:00")
+MONTHLY_STAT_DAY = get_int("MONTHLY_STAT_DAY", "28")
 
 # Параметры рассылки чеков
-MONTHLY_RECEIPT_REMINDER_TIME = get_datetime("MONTHLY_RECEIPT_REMINDER_TIME")
-MONTHLY_RECEIPT_REMINDER_DAY = get_int("MONTHLY_RECEIPT_REMINDER_DAY")
+MONTHLY_RECEIPT_REMINDER_TIME = get_datetime("MONTHLY_RECEIPT_REMINDER_TIME", "12:00")
+MONTHLY_RECEIPT_REMINDER_DAY = get_int("MONTHLY_RECEIPT_REMINDER_DAY", "20")
 
 # Параметры рассылки напоминаний
 DAILY_COLLECT_CONSULTATIONS_TIME = time(hour=0, minute=0, tzinfo=timezone.utc)
-DAILY_CONSULTATIONS_REMINDER_TIME = get_time("DAILY_CONSULTATIONS_REMINDER_TIME")
+DAILY_CONSULTATIONS_REMINDER_TIME = get_time("DAILY_CONSULTATIONS_REMINDER_TIME", "17:00")
 
 # Файл с сохраненными данными бота
-BOT_PERSISTENCE_FILE = BASE_DIR / "persistence_data" / get_string("BOT_PERSISTENCE_FILE")
+BOT_PERSISTENCE_FILE = BASE_DIR / "persistence_data" / "bot_persistence_file"
```

Comment:
Давай так сделаем, чтобы не усложнять.
```suggestion
BOT_PERSISTENCE_FILE = BASE_DIR / "bot_persistence_file"
``` 
И в докер композ файле нужно синхронизировать пути, так как сейчас вообще не работает и не будет работать.